Makefile

```diff
@@ -2,7 +2,7 @@ NAME = ircserv
 
 CC = c++
 
-CFLAGS = -Wall -Wextra -Werror -std=c++98
+CFLAGS = -Wall -Wextra -Werror -std=c++98 -g3
```

Comment:
It is better to replace CFLAGS bt CXXFLAGS as CXXFLAGS the implicit rule for cpp flags